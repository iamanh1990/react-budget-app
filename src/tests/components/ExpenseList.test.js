import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseList } from '../../components/ExpenseList';
import expenses from '../fixtures/expenses';

test('should render Expense List correctly', () => {
	const wrapper = shallow(<ExpenseList expenses={expenses} />);
	expect(wrapper).toMatchSnapshot();
});
