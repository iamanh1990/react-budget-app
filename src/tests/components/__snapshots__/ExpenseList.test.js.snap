// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense List correctly 1`] = `
<div>
  <ExpenseListItem
    amount={100}
    createdAt={0}
    description="rent bill"
    id="1"
    key="1"
    note=""
  />
  <ExpenseListItem
    amount={500}
    createdAt={-86400000}
    description="water bill"
    id="2"
    key="2"
    note=""
  />
  <ExpenseListItem
    amount={30}
    createdAt={432000000}
    description="cleaning bill"
    id="3"
    key="3"
    note=""
  />
</div>
`;
